broker-core/src/main/java/org/apache/qpid/server/virtualhost/UserConnectionLimitChangeListener.java

```diff
@@ -0,0 +1,78 @@
+package org.apache.qpid.server.virtualhost;
+
+import org.apache.qpid.server.model.AbstractConfigurationChangeListener;
+import org.apache.qpid.server.model.ConfiguredObject;
+import org.apache.qpid.server.model.VirtualHost;
+import org.apache.qpid.server.model.VirtualHostUserConnectionLimitProvider;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+final class UserConnectionLimitChangeListener extends AbstractConfigurationChangeListener
```

Comment:
In case of multiple connection limiters every virtual host needs a master coordinator that keeps the consistency of the counters inside the connection limiters. Hence, the functionality can not be moved. The master coordinator has to be notified about the change.